QMLComponents/controls/expanderbuttonbase.cpp

```diff
@@ -35,7 +35,8 @@ void ExpanderButtonBase::setUp()
 
 QString ExpanderButtonBase::generateMDHelp(int depth) const
 {
-	if (!hasInfo()) return "";
+	if (!hasInfo() || !isVisible()) 
+		return "";
```

Comment:
This is also not good: for example a Section makes the controls invisible, but there should be still a help.